pkg/exasol-rest-api/application_properties.go

```diff
@@ -1,48 +1,61 @@
 package exasol_rest_api
 
 import (
-	"errors"
+	"fmt"
+	error_reporting_go "github.com/exasol/error-reporting-go"
 	"os"
 )
 
-//ApplicationProperties for Exasol REST API service.
+// ApplicationProperties for Exasol REST API service.
 type ApplicationProperties struct {
 	ApplicationServer         string `yaml:"server-address"`
 	ExasolUser                string `yaml:"exasol-user"`
 	ExasolPassword            string `yaml:"exasol-password"`
 	ExasolHost                string `yaml:"exasol-host"`
 	ExasolPort                int    `yaml:"exasol-port"`
-	ExasolWebsocketApiVersion int    `yaml:"exasol-websocket-api-version"`
+	ExasolWebsocketAPIVersion int    `yaml:"exasol-websocket-api-version"`
 	Encryption                bool   `yaml:"encryption"`
 	UseTLS                    bool   `yaml:"use-tls"`
 }
 
-//GetApplicationProperties creates an application properties.
+// GetApplicationProperties creates an application properties.
 func GetApplicationProperties(applicationPropertiesPathKey string) *ApplicationProperties {
 	propertiesPath := os.Getenv(applicationPropertiesPathKey)
 	if propertiesPath == "" {
-		panic("runtime error: missing environment variable: " + applicationPropertiesPathKey)
+		panic(error_reporting_go.ExaError("E-ERA-4").Message("runtime error: missing environment variable: {{env}}.").
+			Parameter("env", applicationPropertiesPathKey).
+			Mitigation("please set the variable according to the user guide.").String())
 	}
+
 	properties, err := readApplicationProperties(propertiesPath)
 	if err != nil {
-		panic("runtime error: application properties are missing or incorrect. " + err.Error())
+		panic(error_reporting_go.ExaError("E-ERA-5").
+			Message("runtime error: application properties are missing or incorrect. {{error|uq}}").
+			Parameter("error", err.Error()).String())
 	}
+
 	return properties
 }
 
 func readApplicationProperties(propertiesFilePath string) (*ApplicationProperties, error) {
 	properties, err := getPropertiesFromFile(propertiesFilePath)
 	if err != nil {
-		return nil, err
+		return nil, error_reporting_go.ExaError("E-ERA-6").
+			Message("cannot read properties from specified file: {{filePath}}. {{error|uq}}").
```

Comment:
```suggestion
			Message("Cannot read properties from specified file: {{filePath}}. {{error|uq}}").
```

Reply:
In java is started to write in words with white space instead (e.g: `file path`).  That's more readable in the catalog. But I think it's also fine like that

Reply:
I also use Java-style words in my other project, but separated works look better, I'll change it here